tethys_apps/urls.py

```diff
@@ -8,46 +8,41 @@
 ********************************************************************************
 """
 import logging
-from django.conf.urls import url, include
+from django.urls import include, re_path
+from channels.routing import URLRouter
 from tethys_apps.harvester import SingletonHarvester
 from tethys_apps.views import library, send_beta_feedback_email
 
 tethys_log = logging.getLogger('tethys.' + __name__)
 
 urlpatterns = [
-    url(r'^$', library, name='app_library'),
-    url(r'^send-beta-feedback/$', send_beta_feedback_email, name='send_beta_feedback'),
+    re_path(r'^$', library, name='app_library'),
+    re_path(r'^send-beta-feedback/$', send_beta_feedback_email, name='send_beta_feedback'),
 ]
 
 # Append the app urls urlpatterns
 harvester = SingletonHarvester()
 normal_url_patterns = harvester.get_url_patterns()
 handler_url_patterns = harvester.get_handler_patterns()
 app_url_patterns = normal_url_patterns['app_url_patterns']
```

Comment:
Is `app_url_patterns` used anywhere else other than the loop below? If not we should inline it in the loop as we do with `http_handler_patterns` for consistency.